### ISSUE TITLE
How does "Continue with" appear on the login interface?

### ISSUE THREAD

**Issue by qq1176914912** [casdoor/casdoor#2445]
The website I visited is“ https://conf.casbin.com/ ”I found that when I log in, then log out, and log in again, the login page will display "Continue with", as shown in the following figure:
![image](https://github.com/casdoor/casdoor/assets/79817742/eea2bced-7a0c-400c-8852-316d6617059e)
At that time, the demonstration website“ https://demo.casdoor.com/ ”Why does the same operation not occur? What is the principle of 'Continue with'?

**Comment by hsluoyz** [comment]
@qq1176914912 enable `Signin session` switch in application edit page

![image](https://github.com/casdoor/casdoor/assets/3787410/5d252473-a882-438a-bd25-a34bb168117b)

**Comment by qq1176914912** [comment]
> @qq1176914912 enable `Signin session` switch in application edit page
> 
> ![image](https://user-images.githubusercontent.com/3787410/278311955-5d252473-a882-438a-bd25-a34bb168117b.png)

Is this feature related to cookies? I used casdoor to log in. After logging in to my client and clicking to exit, I logged in again and did not see 'Continue with'. Is it because my client cleared the cookies? Or is it because my client completely disconnected this session?

**Comment by hsluoyz** [comment]
@qq1176914912 "Continue with" relies on cookie. Casdoor relies on cookie to recognize the user and show "Continue with". You need to enable `Signin session` to tell Casdoor to remember the cookie when logged in

**Comment by qq1176914912** [comment]
> @qq1176914912 "Continue with" relies on cookie. Casdoor relies on cookie to recognize the user and show "Continue with". You need to enable `Signin session` to tell Casdoor to remember the cookie when logged in

Yes, I have enabled 'Signin session', but 'Continue with' still does not appear. I have noticed that if I delete my client's token on the casdoor during my client login period, and then click on the exit button in the client section, an error will be reported and the token cannot be found. When refreshing my client page, it will automatically jump to the login page, and 'Continue with' will only appear at this time, I don't know where the problem is.
